fixtures/parallelizer/__init__.py

```diff
@@ -681,6 +694,80 @@ def _modscope_id_splitter(self, module_items):
             self.log.info('sent tests with param %s %r' % (id, tests))
             yield tests
 
+    def get(self, slave):
+        with self.pool_lock:
+            if not self._pool:
+                for test_group in self.test_groups:
+                    self._pool.append(test_group)
+                    for test in test_group:
+                        if '[' in test:
+                            found_prov = []
+                            for pv in self.provs:
+                                if pv in test:
+                                    found_prov.append(pv)
+                                    break
```

Comment:
So this finds only the first one and then breaks to prevent confusion of eg. `nested-rhevm` and `rhevm`?